src/org/starexec/app/RESTServices.java

```diff
@@ -2855,12 +2855,12 @@ public String copySolversToSpace(@PathParam("spaceId") int spaceId, @Context Htt
 			// Get the id of the user who initiated the request
 			int requestUserId = SessionUtil.getUserId(request);
 
-			// Get the space the solver is being copied from
 			String fromSpace = request.getParameter("fromSpace");
-			log.debug("fromSpace: " + fromSpace);
 			Integer fromSpaceId=null;
 			//if null, we are not copying from anywhere-- we are just putting a solver into a new space
 			if (fromSpace!=null) {
+				// Get the space the solver is being copied from
+				log.debug("fromSpace: " + fromSpace);

```

Comment:
This change is mostly unrelated to the PR, but seemed like a worthy change as I was going through the code.